pillar.example

```diff
@@ -164,3 +165,15 @@ prometheus:
     # 'Alternatives system' priority: zero disables (default)
     # yamllint disable-line rule:braces
     altpriority: {{ range(1, 9100000) | random }}
+
+  node_exporter:
```

Comment:
For consistency please move `prometheus.node_exporter` to `prometheus.config.node_exporter` at minimum.  If `exporters` are a valid prometheus services sub-category then we could consider `prometheus.config.exporters.node_exporter` but I'm not that familiar with prometheus so unsure which makes best sense from 1000m viewpoint.  But its inconsistent right now.

Reply:
As I said, `prometheus:config:node_exporter` already has its purpose. I won't interfere with that.
`prometheus:exporters:node_exporter` looks good though.

Reply:
Currently `prometheus:config:node_exporter` is not defined.  I would like `prometheus:node_exporter` do be moved to `prometheus:config:node_exporter` so that your PR defines the dict.

*current pillar data design*
```
prometheus:
  wanted:
    - prometheus
    - alertmanager
    - node_explorer
  pkg:
    prometheus: { .. }
    alertmanager: { ..}
  service:
     prometheus: { ..}
     alertmanager: {..}
     node_explorer: {..}
  config:
     prometheus: { ..}
     alertmanager: {..}
     node_explorer: {..}
```
Otherwise the pillar data looks inconsistent...

*proposed pillar.example design*
```
prometheus:
  wanted:
    - prometheus
    - alertmanager
    - node_explorer
  pkg:
    prometheus: { .. }
    alertmanager: { ..}
  service:
     prometheus: { ..}
     alertmanager: {..}
     node_explorer: {..}
  config:
     prometheus: { ..}
     alertmanager: {..}
  node_explorer: {..}
```

Reply:
> `prometheus:exporters:node_exporter` looks good though.

Hi @alxwr I looked at [prometheus architecture](https://prometheus.io/docs/introduction/overview/#architecture) and this probably is best solution.  There are many exporters so a dedicated dict makes sense. Could you do that?

Reply:
To be clear: `prometheus:exporters:node_exporter`